app/static/pages/home/situation_description/volunteer_situation.html

```diff
@@ -0,0 +1,12 @@
+<p class="oppia-foundation-bold-text">
+    Ideal for...
+</p>
+  <ul>
```

Comment:
Deindent this and the following lines by 2. Ditto elsewhere.

Reply:
Done here and the the two other HTML templates.